react/components/ButtonLink.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent, ReactChild, ReactNode } from 'react'
+import { Button, ButtonWithIcon } from 'vtex.styleguide'
+
+interface Props {
+  url: string
+  icon?: ReactNode
+  fullWidth?: boolean
+  openNewWindow?: boolean
+  variation: string
+  children: ReactChild
+}
+
+const ButtonLink: FunctionComponent<Props> = ({
+  url,
+  icon,
+  fullWidth,
+  openNewWindow,
+  variation,
+  children,
+}) => (
+  <a href={url} target={openNewWindow ? '_blank' : ''}>
```

Comment:
Why not use `<Link>` component from render?

Reply:
No particular reason. Updated it to use `<Link>`, but I could not find the option to pass a `target` argument to it, maybe open an Issue at `render-runtime`?